src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,16 +1,17 @@
 package mate.academy.service;
 
+import mate.academy.model.User;
+
 public class AuthenticationService {
-    /**
-     * Imagine that some user wants to login to your site.
-     * You should check if user credentials (login and password) are valid or not.
-     * All users are stored in <code>UserService</code> class.
-     * @param email - user's email
-     * @param password - user's password
-     * @return true if user by email exists and passed password is equal to user's password.
-     * Return false in any other cases.
-     */
```

Comment:
I would recommend not to remove java documentation, as it generally helps when reading somebody's else code